docs/tutorials/standalone/Soil/evaporation_gilat_loess.jl

```diff
@@ -0,0 +1,356 @@
+ν_ss_quartz = FT(0.3)
+ν_ss_gravel = FT(0.0)
+emissivity = FT(1.0)
+PAR_albedo = FT(0.2)
+NIR_albedo = FT(0.4)
+z_0m = FT(1e-3)
+z_0b = FT(1e-4)
+d_ds = FT(0.01)# 10mm
+params = ClimaLand.Soil.EnergyHydrologyParameters(
+    FT;
+    ν,
+    ν_ss_om,
+    ν_ss_quartz,
+    ν_ss_gravel,
+    hydrology_cm = hcm,
+    K_sat,
+    S_s,
+    θ_r,
+    PAR_albedo,
+    NIR_albedo,
+    emissivity,
+    z_0m,
+    z_0b,
+    earth_param_set,
+    d_ds,
+);
+
+# Boundary conditions
+# No evaporation
+zero_water_flux = WaterFluxBC((p, t) -> 0)
+zero_heat_flux = HeatFluxBC((p, t) -> 0)
+no_flux_boundary_fluxes = (;
+    top = WaterHeatBC(; water = zero_water_flux, heat = zero_heat_flux),
+    bottom = WaterHeatBC(; water = zero_water_flux, heat = zero_heat_flux),
+);
+# With evaporation
+ref_time = DateTime(2005)
+SW_d = (t) -> 0
+LW_d = (t) -> 294.15^4 * 5.67e-8
+radiation = PrescribedRadiativeFluxes(
+    FT,
+    TimeVaryingInput(SW_d),
+    TimeVaryingInput(LW_d),
+    ref_time,
+)
+# Atmos
+T_air = FT(301.15)
+rh = FT(0.38)
+esat = Thermodynamics.saturation_vapor_pressure(
+    thermo_params,
+    T_air,
+    Thermodynamics.Liquid(),
+)
+e = rh * esat
+q = FT(0.622 * e / (101325 - 0.378 * e))
+precip = (t) -> 0.0
+T_atmos = (t) -> T_air
+u_atmos = (t) -> 1.0
+q_atmos = (t) -> q
+h_atmos = FT(0.1)
+P_atmos = (t) -> 101325
+gustiness = FT(1e-2)
+atmos = PrescribedAtmosphere(
+    TimeVaryingInput(precip),
+    TimeVaryingInput(precip),
+    TimeVaryingInput(T_atmos),
+    TimeVaryingInput(u_atmos),
+    TimeVaryingInput(q_atmos),
+    TimeVaryingInput(P_atmos),
+    ref_time,
+    h_atmos;
+    gustiness = gustiness,
+)
+top_bc = ClimaLand.Soil.AtmosDrivenFluxBC(atmos, radiation)
+evap_boundary_fluxes = (;
+    top = top_bc,
+    bottom = WaterHeatBC(; water = zero_water_flux, heat = zero_heat_flux),
+)
+
+# Without drainage - wet boundary at bottom
+bottom_water_bc = MoistureStateBC((p, t) -> 0.35)
+no_drainage_boundary_fluxes = (;
+    top = top_bc,
+    bottom = WaterHeatBC(; water = bottom_water_bc, heat = zero_heat_flux),
+)
+
+# Simulation setup - no evaporation
+t0 = Float64(0)
+tf = Float64(24 * 3600 * 15)
+dt = Float64(10.0)
+Δz = 0.01
+zmax = FT(0)
+zmin = FT(-1.6)
+nelems = Int((zmax - zmin) / Δz)
+soil_domain = Column(; zlim = (zmin, zmax), nelements = nelems)
+z = ClimaCore.Fields.coordinate_field(soil_domain.space.subsurface).z
+
+soil = Soil.EnergyHydrology{FT}(;
+    parameters = params,
+    domain = soil_domain,
+    boundary_conditions = no_flux_boundary_fluxes,
+    sources = (),
+);
+# Initial conditions
+
+Y, p, cds = initialize(soil)
+function estimated_ic(z)
+    0.34 / (1 + exp(-(z + 0.165) / 0.005)) + 0.05
+end
+function init_soil!(Y, z, params)
+    FT = eltype(Y.soil.ϑ_l)
+    Y.soil.ϑ_l .= estimated_ic.(z)
+    Y.soil.θ_i .= 0
+    T = FT(294.15)
+    ρc_s = @. Soil.volumetric_heat_capacity(Y.soil.ϑ_l, FT(0), params)
+    Y.soil.ρe_int =
+        Soil.volumetric_internal_energy.(FT(0), ρc_s, T, Ref(params))
```

Comment:
Is the `FT(0)` in these two lines `Ksat`? If so, it would be good to set it on another line to clarify

Reply:
I defined it in another line, good call!